src/main/java/org/openstreetmap/josm/plugins/pt_assistant/actions/PTMendRelationSupportClasses.java

```diff
@@ -0,0 +1,447 @@
+            if (numeric)
+                chr = '1';
+
+            if (showOption0 && numeric) {
+                if (!shorterRoutes)
+                    drawFixVariantLetter("0 : " + tr(I18N_TURN_BY_TURN_NEXT_INTERSECTION), Color.ORANGE, letterX,
+                        letterY, 25);
+                else
+                    drawFixVariantLetter("0 : " + tr(I18N_SOLUTIONS_BASED_ON_OTHER_RELATIONS), Color.PINK, letterX,
+                        letterY, 25);
+                letterY = letterY + 60;
+            } else if (showOption0) {
+                if (!shorterRoutes)
+                    drawFixVariantLetter("W : " + tr(I18N_TURN_BY_TURN_NEXT_INTERSECTION), Color.ORANGE, letterX,
+                        letterY, 25);
+                else
+                    drawFixVariantLetter("W : " + tr(I18N_SOLUTIONS_BASED_ON_OTHER_RELATIONS), Color.PINK, letterX,
+                        letterY, 25);
+                letterY = letterY + 60;
+            }
+
+            for (int i = 0; i < 5; i++) {
+                if (wayColoring.containsValue(chr)) {
+                    drawFixVariantLetter(chr.toString(), colors[i], letterX, letterY, 35);
+                    letterY = letterY + 60;
+                }
+                chr++;
+            }
+
+            /* display the "Esc", "Skip" label: */
+            drawFixVariantLetter("Esc : " + tr(I18N_CLOSE_OPTIONS), Color.WHITE, letterX, letterY, 25);
+
+            letterY = letterY + 60;
+            if (numeric) {
+                drawFixVariantLetter("7 : " + tr(I18N_SKIP), Color.WHITE, letterX, letterY, 25);
+                letterY = letterY + 60;
+                drawFixVariantLetter("8 : " + tr(I18N_BACKTRACK_WHITE_EDGE), Color.WHITE, letterX, letterY, 25);
+                letterY = letterY + 60;
+                drawFixVariantLetter("9 : " + tr(I18N_REMOVE_CURRENT_EDGE), Color.WHITE, letterX, letterY, 25);
+            } else {
+                drawFixVariantLetter("S : " + tr(I18N_SKIP), Color.WHITE, letterX, letterY, 25);
+                letterY = letterY + 60;
+                drawFixVariantLetter("V : " + tr(I18N_BACKTRACK_WHITE_EDGE), Color.WHITE, letterX, letterY, 25);
+                letterY = letterY + 60;
+                drawFixVariantLetter("Q : " + tr(I18N_REMOVE_CURRENT_EDGE), Color.WHITE, letterX, letterY, 25);
+            }
+        }
+
+        /**
+         *
+         */
+        public void drawOptionsToRemoveWays(String notice) {
+            drawFixVariantsWithParallelLines(false);
+            boolean numeric = PTAssistantPluginPreferences.NUMERICAL_OPTIONS.get();
+
+            double letterX = MainApplication.getMap().mapView.getBounds().getMinX() + 20;
+            double letterY = MainApplication.getMap().mapView.getBounds().getMinY() + 100;
+
+            if (notice != null) {
+                drawFixVariantLetter("Error:  " + notice, Color.WHITE, letterX, letterY, 25);
+                letterY = letterY + 60;
+            }
+            if (numeric) {
```

Comment:
To simplify things like this if-else:

Put the part that is different for the two cases into a method, something like `getKeyCharacter(final int index)` which checks `PTAssistantPluginPreferences.NUMERICAL_OPTIONS.get()` and returns the character to use:
`getKeyCharacter(0)` returns `1` or `A`
`getKeyCharacter(1)` returns `2` or `B`
…

Then you could use it here like `drawFixVariantLetter(getKeyCharacter(0) + " : " + tr(I18N_REMOVE_WAYS), FIVE_COLOR_PALETTE[0], letterX, letterY, 25)`

Note that currently the `I18N_ADD_ONEWAY_VEHICLE_NO_TWO_WAY` option is always yellow. But in numeric mode it gets the fourth number (`4`), whereas in non-numeric mode it gets the third letter (`C`).
But I think it should be no problem to make it always the third option and make `I18N_REMOVE_WAYS_WITH_PREVIOUS_WAY` always the fourth option.